### PR TITLE
Adding Compass integration docs

### PR DESCRIPTION
**Pull request by parrishioner**
Atlassian has created an integration between its Compass product and LaunchDarkly. These are the integration docs. Hopefully we've got everything in the right place. :)

### CODE REVIEW

**Comment by mmrj** [comment]
Thank you for submitting! We'll take a look once https://github.com/launchdarkly/integration-framework/pull/16 is approved.

**Comment by parrishioner** [comment]
Thanks @mmrj ! That other PR is now approved and merged 👍

File: src/content/topics/integrations/more/compass.mdx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+---
+
+<Callout intent="primary">
+<CalloutTitle>The Compass integration is a Pro and Enterprise feature</CalloutTitle>
+<CalloutDescription>
+
+The Compass integration is available to customers on a Pro or Enterprise plan. To learn more, [read about our pricing](https://launchdarkly.com/pricing/). To upgrade your plan, [contact Sales](https://launchdarkly.com/contact-sales/).
+
+</CalloutDescription>
+</Callout>
```
Comment (mmrj) [inline src/content/topics/integrations/more/compass.mdx:15]:
```suggestion
<Callout intent="primary">
  <CalloutTitle>The Compass integration is a Pro and Enterprise feature</CalloutTitle>
  <CalloutDescription>

  The Compass integration is available to customers on a Pro or Enterprise plan. To learn more, [read about our pricing](https://launchdarkly.com/pricing/). To upgrade your plan, [contact Sales](https://launchdarkly.com/contact-sales/).

  </CalloutDescription>
</Callout>
```
The callout is a custom component we built, and it's pretty finicky about whitespace. This edit, and the similar ones below, just ensure the component builds and displays as intended. Here's a screenshot from my local build of your branch to confirm: 

callouts after whitespace edits

File: src/content/topics/integrations/more/compass.mdx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+This topic explains how to use the Compass integration for LaunchDarkly.
+
+<Callout intent="info">
+<CalloutTitle>The Compass integration is managed by Atlassian</CalloutTitle>
+<CalloutDescription>
+
+The Compass integration was built by and is managed by Atlassian. For questions about the integration or for technical support, [contact Atlassian](https://community.atlassian.com/t5/Compass-Alpha/gh-p/compass-alpha).
+
+</CalloutDescription>
+</Callout>
```
Comment (mmrj) [inline src/content/topics/integrations/more/compass.mdx:28]:
```suggestion
<Callout intent="info">
  <CalloutTitle>The Compass integration is managed by Atlassian</CalloutTitle>
  <CalloutDescription>

  The Compass integration was built by and is managed by Atlassian. For questions about the integration or for technical support, [contact Atlassian](https://community.atlassian.com/t5/Compass-Alpha/gh-p/compass-alpha).

  </CalloutDescription>
</Callout>
```
whitespace only

File: src/content/topics/integrations/more/compass.mdx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+The Compass integration for LaunchDarkly displays a timeline of feature flag changes related to a Compass component, as well as a complete list of feature flags and their current state. This helps you visualize how recent changes to feature flags may be impacting component health.
+
+## Prerequisites
+
+To connect Compass to LaunchDarkly, you must meet the following prerequisites:
+
+1. Be a Compass administrator on your Atlassian site
```
Comment (mmrj) [inline src/content/topics/integrations/more/compass.mdx:36]:
```suggestion
1. Be a Compass administrator on your Atlassian site.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+2. Generate a LaunchDarkly [API access token](/home/account-security/api-access-tokens) for use with Compass.
+* Create a token with the **Reader** role.
+* If you are on an Enterprise plan, check the **This is a service token** option when creating your acesss token.
+* If you are on a Pro plan, leave this option unchecked.
+* Be sure to copy the new token to your clipboard.
```
Comment (mmrj) [inline src/content/topics/integrations/more/compass.mdx:41]:
```suggestion
2. Have a LaunchDarkly [API access token](/home/account-security/api-access-tokens) for use with Compass. To generate a token:
* Create a token with the **Reader** role.
* If you are on an Enterprise plan, check the **This is a service token** option when creating your access token.
* If you are on a Pro plan, leave this option unchecked.
* Be sure to copy the new token and save it somewhere secure.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+
+## Setting up the Compass integration
+
+The Compass integration is managed from the Compass product on your Atlassian Cloud site. Other than creating an API access token, there are no Compass settings or preferences to configure in LaunchDarkly.
+
+1. Generate a LaunchDarkly API access token as described in the [Prerequisites](#prerequisites) section.
```
Comment (mmrj) [inline src/content/topics/integrations/more/compass.mdx:47]:
```suggestion
The Compass integration is managed from the Compass product on your Atlassian Cloud site. Other than creating an API access token, there are no Compass settings or preferences to configure in LaunchDarkly.

Here's how to set up your Compass integration:

1. Generate a LaunchDarkly API access token as described in the [Prerequisites](#prerequisites) section.
```

We typically introduce each procedure with a short stem sentence to be explicit about exactly what the procedure is doing.

File: src/content/topics/integrations/more/compass.mdx [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+2. Go to your Atlassian Compass site, e.g. https://mysite.atlassian.net/compass.
```
Comment (mmrj) [inline src/content/topics/integrations/more/compass.mdx:48]:
```suggestion
2. Go to your Atlassian Compass site, for example, `https://mysite.atlassian.net/compass`.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+7. On the **LaunchDarkly is requesting Access** screen, review the requested permissions, then click the **Accept** button.
+
+<Callout intent="info">
+<CalloutTitle>You must authorize the requested permissions for the integration to work</CalloutTitle>
+<CalloutDescription>
+
+If you click **Cancel**, the integration cannot access your Compass data.
+
+</CalloutDescription>
+</Callout>
```
Comment (mmrj) [inline src/content/topics/integrations/more/compass.mdx:62]:
```suggestion
<Callout intent="info">
  <CalloutTitle>You must authorize the requested permissions for the integration to work</CalloutTitle>
  <CalloutDescription>

  If you click **Cancel**, the integration cannot access your Compass data.

  </CalloutDescription>
</Callout>
```
whitespace only

File: src/content/topics/integrations/more/compass.mdx [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+
+The Compass integration is managed from the Compass product on your Atlassian Cloud site. Other than creating an API access token, there are no Compass settings or preferences to configure in LaunchDarkly.
+
+1. Generate a LaunchDarkly API access token as described in the [Prerequisites](#prerequisites) section.
+2. Go to your Atlassian Compass site, e.g. https://mysite.atlassian.net/compass.
+3. In the top nagivation, click the **Apps** link.
+4. Find **LaunchDarkly** in the list of apps and click the **Install** button.
+5. After the LaunchDakly app is installed, click the **Configure** button.
+6. Click the **Allow access** button to approve the access to your Atlassian account requested by the LaunchDarkly app.
+7. On the **LaunchDarkly is requesting Access** screen, review the requested permissions, then click the **Accept** button.
```
Comment (mmrj) [inline src/content/topics/integrations/more/compass.mdx:53]:
```suggestion
3. In the top navigation, click the **Apps** link.
4. Find **LaunchDarkly** in the list of apps and click **Install**.
5. After the LaunchDakly app is installed, click **Configure**.
6. Click **Allow access** to approve the access to your Atlassian account requested by the LaunchDarkly app.
7. On the **LaunchDarkly is requesting Access** screen, review the requested permissions, then click **Accept**.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 9]
```diff
@@ -0,0 +1,144 @@
...
+<CalloutDescription>
+
+If you click **Cancel**, the integration cannot access your Compass data.
+
+</CalloutDescription>
+</Callout>
+
+![LaunchDarkly authorization screen for Compass](compass-ld-integration-auth-dialog.png)
+
+8. On the app configuration screen, paste your LaunchDarkly API access token and click the **Submit** button.
```
Comment (mmrj) [inline src/content/topics/integrations/more/compass.mdx:66]:
```suggestion
8. On the app configuration screen, paste your LaunchDarkly API access token and click **Submit**.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 10]
```diff
@@ -0,0 +1,144 @@
...
+
+## Connecting Compass components to LaunchDarkly
+
+Now that you've set up the Compass integration for Compass, you must conncet your LaunchDarkly projects to Compass components in order to see feature flag data in Compass.
+
+To connect a LaunchDarkly project to a Compass component:
+
+1. In LaunchDarkly, navigate to the **Feature flags** page for the project you want to connect.
+2. Copy the project URL, e.g. https://app.launchdarkly.com/starship-command/production/features, to your clipboard.
+3. Navigate to your Compass site, e.g. https://mysite.atlassian.net/compass
```
Comment (mmrj) [inline src/content/topics/integrations/more/compass.mdx:82]:
```suggestion
2. Copy the project URL, for example, `https://app.launchdarkly.com/starship-command/production/features`, to your clipboard.
3. Navigate to your Compass site, for example, `https://mysite.atlassian.net/compass`.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 11]
```diff
@@ -0,0 +1,144 @@
...
+
+![Successfully connected Compass component to LaunchDarkly](compass-ld-integration-component-connected.png)
+
+7. Repeat the above steps to associate your other LaunchDarkly projects to the applicable Compass components.
+
+## Viewing feature flag information for a Compass component
+
+Once you have connected your Compass components to LaunchDarkly, you can view feature flag information on the component page in Compass.
+
+1. Navigate to your Compass site, e.g. https://mysite.atlassian.net/compass
```
Comment (mmrj) [inline src/content/topics/integrations/more/compass.mdx:95]:
```suggestion
Once you have connected your Compass components to LaunchDarkly, you can view feature flag information on the component page in Compass.

Here's how:

1. Navigate to your Compass site, for example, `https://mysite.atlassian.net/compass`.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 12]
```diff
@@ -0,0 +1,144 @@
...
+3. Make sure you have completed the steps in [Connecting Compass components to LaunchDarkly](#connecting-compass-components-to-launchdarkly).
+4. Click on the **LaunchDarkly** tab in the left navigation.
+
+### Recent feature flag changes for a connected Compass component
+
+You can see a history of recent feature flag changes for the associated LaunchDarkly project on the **Recently updated** subtab.
+* Click the dot next to each item to see more details about the feature flag change.
+* Click the feature flag name to view the feature flag in LaunchDarkly.
+
+![Recently feature flag changes](compass-ld-integration-recently-updated.png)
```
Comment (mmrj) [inline src/content/topics/integrations/more/compass.mdx:106]:
```suggestion
![Recently updated feature flag displayed in Compass](compass-ld-integration-recently-updated.png)
```

File: src/content/topics/integrations/more/compass.mdx [hunk 13]
```diff
@@ -0,0 +1,144 @@
...
+
+### All feature flags for a connected Compass component
+
+You can see a list of all feature flags for the associated LaunchDarkly project on the **All feature flags** subtab.
+* Use the **Filter by status** dropdown to filter the list of flags displayed in the list
```
Comment (mmrj) [inline src/content/topics/integrations/more/compass.mdx:111]:
```suggestion
* Use the **Filter by status** dropdown to filter the list of flags displayed in the list.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 14]
```diff
@@ -0,0 +1,144 @@
...
+* Click the column headings above the list to sort by the **Flag**, **Status** or **Last requested** column.
+* Click the feature flag name to view the feature flag in LaunchDarkly.
+
+![All feature flags](compass-ld-integration-all-feature-flags.png)
```
Comment (mmrj) [inline src/content/topics/integrations/more/compass.mdx:115]:
```suggestion
![All feature flags displayed in Compass](compass-ld-integration-all-feature-flags.png)
```

File: src/content/topics/integrations/more/compass.mdx [hunk 15]
```diff
@@ -0,0 +1,144 @@
...
+
+## Disconnecting Compass components from LaunchDarkly
+
+The connection between a LaunchDarkly project and a Compass component is stored as a link on the Compass component, as described in the [Connecting Compass components to LaunchDarkly](#connecting-compass-components-to-launchdarkly) section. If desired, you can remove this connection on the Compass component page.
+
+1. Navigate to your Compass site, e.g. https://mysite.atlassian.net/compass
```
Comment (mmrj) [inline src/content/topics/integrations/more/compass.mdx:121]:
```suggestion

Here's how: 

1. Navigate to your Compass site, for example `https://mysite.atlassian.net/compass`.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 16]
```diff
@@ -0,0 +1,144 @@
...
+2. Navigate to the Compass component you want to connect to LaunchDarkly.
```
Comment (mmrj) [inline src/content/topics/integrations/more/compass.mdx:122]:
```suggestion
2. Navigate to the Compass component you want to disconnect from LaunchDarkly.
```
I think update fixes a copy-paste error, but please check this for accuracy! Thanks.

File: src/content/topics/integrations/more/compass.mdx [hunk 17]
```diff
@@ -0,0 +1,144 @@
...
+3. In the **Other links** section of your Compass component, remove the LaunchDarkly project link.
+4. Verify that the component was successfully disconnected by clicking on the **LaunchDarkly** tab in the left navigation. You should no longer see feature flag information on this tab.
+
+## Disconnecting your API access token
+
+The Compass integration stores your LaunchDarkly API access token (as described in the [Setting up the Compass integration](#setting-up-the-compass-integration)) section) to read data from the LaunchDarkly API. If you need to cycle your LaunchDarkly access token, or if you want to connect Compass to a different LaunchDarkly organization, you can disconnect your access token.
```
Comment (mmrj) [inline src/content/topics/integrations/more/compass.mdx:128]:
```suggestion
The Compass integration stores your LaunchDarkly API access token to read data from the LaunchDarkly API, as described in the [Setting up the Compass integration](#setting-up-the-compass-integration)) section. If you need to cycle your LaunchDarkly access token, or if you want to connect Compass to a different LaunchDarkly organization, you can disconnect your access token.

Here's how:
```

File: src/content/topics/integrations/more/compass.mdx [hunk 18]
```diff
@@ -0,0 +1,144 @@
...
+
+1. Go to your Atlassian Compass site, e.g. https://mysite.atlassian.net/compass.
+2. In the top nagivation, click the **Apps** link.
+3. Find **LaunchDarkly** in the list of apps and click the **Configure** button.
+4. Click the **Disconnect access token** button.
```
Comment (mmrj) [inline src/content/topics/integrations/more/compass.mdx:133]:
```suggestion
1. Go to your Atlassian Compass site, for example `https://mysite.atlassian.net/compass`.
2. In the top navigation, click the **Apps** link.
3. Find **LaunchDarkly** in the list of apps and click **Configure**.
4. Click **Disconnect access token**.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 19]
```diff
@@ -0,0 +1,144 @@
...
+5. You can now provide Compass with a new access token if desired.
+
+## Removing the Compass integration
+
+To remove the Compass integration:
+
+1. Go to your Atlassian Compass site, e.g. https://mysite.atlassian.net/compass.
+2. In the top nagivation, click the **Apps** link.
+3. Find **LaunchDarkly** in the list of apps and click the **Uninstall** button.
```
Comment (mmrj) [inline src/content/topics/integrations/more/compass.mdx:142]:
```suggestion
1. Go to your Atlassian Compass site, for example `https://mysite.atlassian.net/compass`.
2. In the top navigation, click the **Apps** link.
3. Find **LaunchDarkly** in the list of apps and click **Uninstall**.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 20]
```diff
@@ -0,0 +1,144 @@
+---
+path: /integrations/compass
+title: Compass
+description: This topic explains visualize LaunchDarkly feature flag changes in Atlassian Compass.
```
Comment (mmrj) [inline src/content/topics/integrations/more/compass.mdx:4]:
```suggestion
description: This topic explains how to use the Compass integration for LaunchDarkly.
```
The description should match the first sentence in the Overview section. Feel free to include something about visualizing feature flag changes if you'd like that in both places.

**Review by mmrj — CHANGES_REQUESTED** [review]
Hi @parrishioner , thanks for submitting integration docs! These look great. The requested changes are minor copyedits and some whitespace updates for one of our custom components. Please let us know if you have any questions or concerns.

File: src/content/topics/integrations/more/compass.mdx [hunk 21]
```diff
@@ -0,0 +1,144 @@
...
+
+You can see a history of recent feature flag changes for the associated LaunchDarkly project on the **Recently updated** subtab.
+* Click the dot next to each item to see more details about the feature flag change.
+* Click the feature flag name to view the feature flag in LaunchDarkly.
+
+![Recently feature flag changes](compass-ld-integration-recently-updated.png)
+
+### All feature flags for a connected Compass component
+
+You can see a list of all feature flags for the associated LaunchDarkly project on the **All feature flags** subtab.
```
Comment (ember-stevens) [inline src/content/topics/integrations/more/compass.mdx:110]:
```suggestion
You can view a list of all feature flags for the associated LaunchDarkly project on the **All feature flags** subtab.
```

File: src/content/topics/integrations/more/compass.mdx (hunk 14 above)
Comment (ember-stevens) [reply src/content/topics/integrations/more/compass.mdx:115]: Also add punctuation at the end.

File: src/content/topics/integrations/more/compass.mdx [hunk 22]
```diff
@@ -0,0 +1,144 @@
...
+
+![LaunchDarkly access token screen for Compass](compass-ld-integration-access-token.png)
+
+9. Confirm that the integration has connected LaunchDarkly and Compass when the **Connected to LaunchDarkly** success message is displayed.
+
+![Successfully connected Compass to LaunchDarkly](compass-ld-integration-connected.png)
+
+## Connecting Compass components to LaunchDarkly
+
+Now that you've set up the Compass integration for Compass, you must conncet your LaunchDarkly projects to Compass components in order to see feature flag data in Compass.
```
Comment (ember-stevens) [inline src/content/topics/integrations/more/compass.mdx:76]:
```suggestion
Now that you've set up the Compass integration for Compass, you must connect your LaunchDarkly projects to Compass components to see feature flag data in Compass.
```

File: src/content/topics/integrations/more/compass.mdx (hunk 12 above)
Comment (ember-stevens) [reply src/content/topics/integrations/more/compass.mdx:106]: Also add punctuation at the end.

**Review by ember-stevens — CHANGES_REQUESTED** [review]

File: src/content/topics/integrations/more/compass.mdx [hunk 23]
```diff
@@ -0,0 +1,144 @@
...
+
+To connect a LaunchDarkly project to a Compass component:
+
+1. In LaunchDarkly, navigate to the **Feature flags** page for the project you want to connect.
+2. Copy the project URL, e.g. https://app.launchdarkly.com/starship-command/production/features, to your clipboard.
+3. Navigate to your Compass site, e.g. https://mysite.atlassian.net/compass
+4. Navigate to the Compass component you want to connect to LaunchDarkly.
+5. In the **Other links** section of your Compass component, paste the LaunchDarkly project URL as a new link.
```
Comment (brandonmensing) [inline src/content/topics/integrations/more/compass.mdx:84]: Per [this conversation](https://launchdarkly.slack.com/archives/C01HCMH33JM/p1649017618030749) we should update this line to be specific about linking to the project and environment.

**Comment by parrishioner** [comment]
Hi @mmrj @ember-stevens @brandonmensing 👋  Thank you for all the comments and suggestions! I've 🤞  hopefully addressed them all and this is ready for review again.

File: src/content/topics/integrations/more/compass.mdx [hunk 24]
```diff
@@ -0,0 +1,152 @@
...
+## Prerequisites
+
+To connect Compass to LaunchDarkly, you must meet the following prerequisites:
+
+* Be a Compass administrator on your Atlassian site.
+* Have a LaunchDarkly [API access token](/home/account-security/api-access-tokens) for use with Compass. To generate a token:
+  * Create a token with the **Reader** role.
+  * If you are on an Enterprise plan, check the **This is a service token** option when creating your access token.
+  * If you are on a Pro plan, leave this option unchecked.
+  * Be sure to copy the new token and save it somewhere secure.
```
Comment (ember-stevens) [inline src/content/topics/integrations/more/compass.mdx:41]:
```suggestion
* Have a LaunchDarkly API access token for use with Compass. To learn more, read [Creating API access tokens](/home/account-security/api-access-tokens#creating-api-access-tokens).

To generate an API token for use with Compass:

1. Create a token with the **Reader** role.
2. If you are on an Enterprise plan, check the **This is a service token** option when creating your access token. If you are on a Pro plan, leave this option unchecked.
3. Copy the new token and save it somewhere secure.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 25]
```diff
@@ -0,0 +1,152 @@
...
+
+## Setting up the Compass integration
+
+The Compass integration is managed from the Compass product on your Atlassian Cloud site. Other than creating an API access token, there are no Compass settings or preferences to configure in LaunchDarkly.
```
Comment (ember-stevens) [inline src/content/topics/integrations/more/compass.mdx:45]:
```suggestion
The Compass integration is managed from the Atlassian Cloud site. Other than creating an API access token, there are no settings or preferences to configure in LaunchDarkly.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 26]
```diff
@@ -0,0 +1,152 @@
...
+
+Here's how to set up your Compass integration:
+
+1. Generate a LaunchDarkly API access token as described in the [Prerequisites](#prerequisites) section.
+2. Go to your Atlassian Compass site, for example, `https://mysite.atlassian.net/compass`.
+3. In the top navigation, click the **Apps** link.
```
Comment (ember-stevens) [inline src/content/topics/integrations/more/compass.mdx:51]:
```suggestion
2. Navigate to your Atlassian Compass site. For example, `https://mysite.atlassian.net/compass`.
3. In the top navigation, click **Apps**.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 27]
```diff
@@ -0,0 +1,152 @@
...
+4. Find **LaunchDarkly** in the list of apps and click **Install**.
+5. After the LaunchDakly app is installed, click **Configure**.
+6. Click **Allow access** to approve the access to your Atlassian account requested by the LaunchDarkly app.
+7. On the **LaunchDarkly is requesting Access** screen, review the requested permissions, then click **Accept**.
```
Comment (ember-stevens) [inline src/content/topics/integrations/more/compass.mdx:55]:
```suggestion
6. Click **Allow access**. The "LaunchDarkly is requesting Access" screen appears.
7. Review the requested permissions, then click **Accept**.

![LaunchDarkly authorization screen for Compass.](compass-ld-integration-auth-dialog.png)
```

File: src/content/topics/integrations/more/compass.mdx [hunk 28]
```diff
@@ -0,0 +1,152 @@
...
+<Callout intent="info">
+  <CalloutTitle>You must authorize the requested permissions for the integration to work</CalloutTitle>
+  <CalloutDescription>
+
+  If you click **Cancel**, the integration cannot access your Compass data.
+
+  </CalloutDescription>
+</Callout>
+
+![LaunchDarkly authorization screen for Compass.](compass-ld-integration-auth-dialog.png)
```
Comment (ember-stevens) [inline src/content/topics/integrations/more/compass.mdx:66]:
```suggestion
```

File: src/content/topics/integrations/more/compass.mdx [hunk 29]
```diff
@@ -0,0 +1,152 @@
...
+
+8. On the app configuration screen, enter your LaunchDarkly API access token and click **Submit**.
```
Comment (ember-stevens) [inline src/content/topics/integrations/more/compass.mdx:68]:
```suggestion
8. Enter your LaunchDarkly API access token in the **Access token** field.
9. Click **Submit**.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 30]
```diff
@@ -0,0 +1,152 @@
...
+
+![LaunchDarkly access token screen for Compass.](compass-ld-integration-access-token.png)
+
+9. Confirm that the integration has connected LaunchDarkly and Compass when the **Connected to LaunchDarkly** success message is displayed.
```
Comment (ember-stevens) [inline src/content/topics/integrations/more/compass.mdx:72]:
```suggestion
The integration has connected LaunchDarkly and Compass when the **Connected to LaunchDarkly** success message is displayed.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 31]
```diff
@@ -0,0 +1,152 @@
...
+
+![Successfully connected Compass to LaunchDarkly.](compass-ld-integration-connected.png)
+
+## Connecting Compass components to LaunchDarkly
+
+Now that you've set up the Compass integration for Compass, you must connect your LaunchDarkly projects to Compass components to see feature flag data in Compass.
```
Comment (ember-stevens) [inline src/content/topics/integrations/more/compass.mdx:78]:
```suggestion
Now that you've set up the integration, you must connect your LaunchDarkly projects to Compass components to see feature flag data in Compass.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 32]
```diff
@@ -0,0 +1,152 @@
...
+
+To connect a LaunchDarkly project to a Compass component:
+
+1. In LaunchDarkly, navigate to the **Feature flags** page for the project you want to connect.
+2. Copy the page URL, for example, `https://app.launchdarkly.com/starship-command/production/features`, to your clipboard. Note that the URL contains the project name and environment.
+3. Navigate to your Compass site, for example, `https://mysite.atlassian.net/compass`.
```
Comment (ember-stevens) [inline src/content/topics/integrations/more/compass.mdx:84]:
```suggestion
2. Copy the page URL to your clipboard. For example, `https://app.launchdarkly.com/starship-command/production/features`. The URL contains the project name and environment.
3. Navigate to your Compass site. For example, `https://mysite.atlassian.net/compass`.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 33]
```diff
@@ -0,0 +1,152 @@
...
+4. Navigate to the Compass component you want to connect to LaunchDarkly.
+5. In the **Other links** section of your Compass component, paste the LaunchDarkly URL as a new link.
+6. Verify that the component was successfully connected by clicking on the **LaunchDarkly** tab in the left navigation.
```
Comment (ember-stevens) [inline src/content/topics/integrations/more/compass.mdx:87]:
```suggestion
6. Click on the **LaunchDarkly** tab in the left navigation to verify that the component was successfully connected.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 34]
```diff
@@ -0,0 +1,152 @@
...
+
+7. Repeat the above steps to associate your other LaunchDarkly projects to the applicable Compass components.
+
+## Viewing feature flag information for a Compass component
+
+Once you have connected your Compass components to LaunchDarkly, you can view feature flag information on the component page in Compass.
+
+Here's how:
+
+1. Navigate to your Compass site, for example, `https://mysite.atlassian.net/compass`.
```
Comment (ember-stevens) [inline src/content/topics/integrations/more/compass.mdx:99]:
```suggestion
1. Navigate to your Compass site. For example, `https://mysite.atlassian.net/compass`.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 35]
```diff
@@ -0,0 +1,152 @@
...
+2. Navigate to the Compass component for which you want to view feature flag information.
+3. Make sure you have completed the steps in [Connecting Compass components to LaunchDarkly](#connecting-compass-components-to-launchdarkly).
+4. Click on the **LaunchDarkly** tab in the left navigation.
```
Comment (ember-stevens) [inline src/content/topics/integrations/more/compass.mdx:102]:
```suggestion
3. Click on the **LaunchDarkly** tab in the left navigation.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 36]
```diff
@@ -0,0 +1,152 @@
...
+
+![All feature flags displayed in Compass.](compass-ld-integration-all-feature-flags.png)
+
+## Disconnecting Compass components from LaunchDarkly
+
+The connection between a LaunchDarkly project and a Compass component is stored as a link on the Compass component, as described in the [Connecting Compass components to LaunchDarkly](#connecting-compass-components-to-launchdarkly) section. If desired, you can remove this connection on the Compass component page.
+
+Here's how: 
+
+1. Navigate to your Compass site, for example `https://mysite.atlassian.net/compass`.
```
Comment (ember-stevens) [inline src/content/topics/integrations/more/compass.mdx:127]:
```suggestion
1. Navigate to your Compass site. For example `https://mysite.atlassian.net/compass`.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 37]
```diff
@@ -0,0 +1,152 @@
...
+2. Navigate to the Compass component you want to disconnect from LaunchDarkly.
+3. In the **Other links** section of your Compass component, remove the LaunchDarkly link.
+4. Verify that the component was successfully disconnected by clicking on the **LaunchDarkly** tab in the left navigation. You should no longer see feature flag information on this tab.
```
Comment (ember-stevens) [inline src/content/topics/integrations/more/compass.mdx:130]:
```suggestion
Verify that the component was successfully disconnected by clicking on the **LaunchDarkly** tab in the left navigation. You should no longer see feature flag information on this tab.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 38]
```diff
@@ -0,0 +1,152 @@
...
+
+## Disconnecting your API access token
+
+The Compass integration stores your LaunchDarkly API access token to read data from the LaunchDarkly API, as described in the [Setting up the Compass integration](#setting-up-the-compass-integration)) section. If you need to cycle your LaunchDarkly access token, or if you want to connect Compass to a different LaunchDarkly organization, you can disconnect your access token.
```
Comment (ember-stevens) [inline src/content/topics/integrations/more/compass.mdx:134]:
```suggestion
The Compass integration stores your LaunchDarkly API access token to read data from the LaunchDarkly API, as described in the [Setting up the Compass integration](#setting-up-the-compass-integration) section. If you need to cycle your LaunchDarkly access token, or if you want to connect Compass to a different LaunchDarkly organization, you can disconnect your access token.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 39]
```diff
@@ -0,0 +1,152 @@
...
+
+Here's how:
+
+1. Go to your Atlassian Compass site, for example `https://mysite.atlassian.net/compass`.
```
Comment (ember-stevens) [inline src/content/topics/integrations/more/compass.mdx:138]:
```suggestion
1. Open to your Atlassian Compass site. For example `https://mysite.atlassian.net/compass`.
```

File: src/content/topics/integrations/more/compass.mdx [hunk 40]
```diff
@@ -0,0 +1,152 @@
...
+3. Find **LaunchDarkly** in the list of apps and click **Configure**.
+4. Click **Disconnect access token**.
+5. You can now provide Compass with a new access token if desired.
+
+## Removing the Compass integration
+
+To remove the Compass integration:
+
+1. Go to your Atlassian Compass site, for example `https://mysite.atlassian.net/compass`.
+2. In the top navigation, click the **Apps** link.
```
Comment (ember-stevens) [inline src/content/topics/integrations/more/compass.mdx:149]:
```suggestion
1. Open your Atlassian Compass site. For example `https://mysite.atlassian.net/compass`.
2. In the top navigation, click **Apps**.
```

**Review by ember-stevens — CHANGES_REQUESTED** [review]
This is looking great! Just lots of little fiddly style tweaks.

File: src/content/topics/integrations/more/compass.mdx (hunk 28 above)
Comment (parrishioner) [reply src/content/topics/integrations/more/compass.mdx:70]: @ember-stevens I agree this image is probably unnecessary. I'll remove the png from the changeset.

**Comment by parrishioner** [comment]
@ember-stevens Thanks for the careful review! Should all be addressed now.

**Review by ember-stevens — APPROVED** [review]

**Comment by mmrj** [comment]
@parrishioner Thank you! This has now been deployed and is available here: https://docs.launchdarkly.com/integrations/compass

During merge/deploy/testing, we realized that the image "compass-ld-integration-auth-dialog.png", on line 59, is not available. Apologies for missing that during the PR review! For the moment we've commented out the image. Could you please open up another PR with the missing image, if you'd still like to include it? Thank you!

**Comment by parrishioner** [comment]
Nice catch @mmrj ! This is addressed in https://github.com/launchdarkly/LaunchDarkly-Docs/pull/258

**Pull request merged**